docs/painless/painless-general-syntax.asciidoc

```diff
@@ -1,5 +1,5 @@
-[[painless-syntax]]
-=== Painless Syntax
+[[painless-general-syntax]]
+=== General Syntax
 
 [float]
```

Comment:
These subheadings should not be floated. That way, they'll show up in the "On this Page" section like the subheadings in the Basic Syntax section.

Reply:
Fixed.